src/engine/PiecewiseLinearConstraint.cpp

```diff
@@ -17,7 +17,8 @@
 #include "Statistics.h"
 
 PiecewiseLinearConstraint::PiecewiseLinearConstraint()
-    : _constraintActive( true )
+    : _temporary( false )
```

Comment:
This _temporary thing is a bit messy. Instead, lets pass the DisjunctionConstraint by value. It's very light weight, so no problem if it gets copied a couple of times.

Reply:
Create a field variable in SmtCore to store this constraint.